src/Adapter/MSTestAdapter.PlatformServices/PublicAPI/net9.0-windows10.0.17763.0/PublicAPI.Shipped.txt

```diff
@@ -1 +1,2 @@
 #nullable enable
+#nullable enableMicrosoft.VisualStudio.TestPlatform.MSTestAdapter.PlatformServices.TraceListenerManager.Close(Microsoft.VisualStudio.TestPlatform.MSTestAdapter.PlatformServices.Interface.ITraceListener! traceListener) -> voidMicrosoft.VisualStudio.TestPlatform.MSTestAdapter.PlatformServices.TraceListenerWrapper.Close() -> voidMicrosoft.VisualStudio.TestPlatform.MSTestAdapter.PlatformServices.TraceListenerWrapper.Dispose() -> void
```

Comment:
@Evangelink Something was still wrong here

Reply:
Yeah I saw on another PR. I'll need to ask @nohwnd because that's out of my PS skills.